debian/rules

```diff
@@ -1,3 +1,20 @@
 #!/usr/bin/make -f
+
+DEB_BUILD_OPTIONS=nocheck
+
 %:
 	dh $@
+
+override_dh_auto_install:
```

Comment:
Is the override for dh_auto_install here because it can't be generalised in the autotools configuration? Or is it too debian-specific to include in the base installation file list.